Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceListener.java

```diff
@@ -80,7 +80,8 @@ public void received(@NotNull final String streamId, @NotNull final Event event)
         break;
       case PausePostRequest:
         // We get this event after an isolate reload call, when pause after reload has been requested.
-        // TODO: Set current breakpoints; resume the isolate.
+        // This adds the "supports.pausePostRequest" capability.
+        myDebugProcess.getVmServiceWrapper().restoreBreakpointsForIsolate(event.getIsolate().getId());
```

Comment:
Re: the shared logic - I would prefer to have the two different variants, as what they do is slightly different (set all and resume the isolate, vs remove and re-set all).